test/unittests/execute_floating_point_test.cpp

```diff
@@ -1360,6 +1372,10 @@ TEST(execute_floating_point, f64_add_off_by_one)
     _FPU_SETCW(fpu_cw);
     */
 
+#if defined(__i386__) && !defined(__SSE_MATH__)
```

Comment:
why not at the top of the test?

Reply:
To execute as much as possible. Can be at the top too.

Reply:
On top it would be more clear that long comment there explains the reason for skipping IMHO.

Reply:
I agree. Will do it.